app/experimenter/nimbus-ui/src/components/TableMetricSecondary/index.test.tsx

```diff
@@ -53,7 +57,9 @@ describe("TableMetricSecondary", () => {
       <RouterSlugProvider mocks={[mock]}>
         <TableMetricSecondary
           results={mockAnalysis().overall}
-          probeSet={experiment.primaryProbeSets![0]!}
+          probeSetSlug={experiment!.secondaryProbeSets![0]!.slug}
```

Comment:
Jody updated the experiment mock where we no longer have to tell TS it's not falsey. You should be able to update all instances of `experiment!` to just `experiment` in your tests/stories.